tests/connect/test_analyze_plan.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
+
+
+def test_analyze_plan(spark_session):
+    data = [[1000, 99]]
+    df1 = spark_session.createDataFrame(data, schema="Value int, Total int")
+    s = df1.schema
+    assert str(s) == "StructType([StructField('_1', LongType(), True), StructField('_2', LongType(), True)])"
```

Comment:
this is a bad assertion. 

`int` should map to [IntegerType](https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.types.IntegerType.html#pyspark.sql.types.IntegerType) not `LongType`

Reply:
ok also seems like it removes names or something? even though what is being returned is 

```
Schema: Schema {
    schema: Some(
        DataType {
            kind: Some(
                Struct(
                    Struct {
                        fields: [
                            StructField {
                                name: "Value",
                                data_type: Some(
                                    DataType {
                                        kind: Some(
                                            Integer(
                                                Integer {
                                                    type_variation_reference: 0,
                                                },
                                            ),
                                        ),
                                    },
                                ),
                                nullable: true,
                                metadata: None,
                            },
                            StructField {
                                name: "Total",
                                data_type: Some(
                                    DataType {
                                        kind: Some(
                                            Integer(
                                                Integer {
                                                    type_variation_reference: 0,
                                                },
                                            ),
                                        ),
                                    },
                                ),
                                nullable: true,
                                metadata: None,
                            },
                        ],
                        type_variation_reference: 0,
                    },
                ),
            ),
        },
    ),
}
```

Reply:
meh yea spark is sparking I think 😅